tests/ballerina-test/src/test/resources/test-src/runtime/invocation-context.bal

```diff
@@ -43,3 +43,14 @@ function testAuthToken () returns (boolean) {
     runtime:getInvocationContext().authContext.authToken = authToken;
     return authToken == runtime:getInvocationContext().authContext.authToken;
 }
+
+function testAttributes () returns (boolean) {
```

Comment:
Brackets around boolean can be removed

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/9069.